src/Microsoft.AspNetCore.Antiforgery/AntiforgeryValidationException.cs

```diff
@@ -19,5 +19,16 @@ public AntiforgeryValidationException(string message)
             : base(message)
         {
         }
+
+        /// <summary>
+        /// Creates a new instance of <see cref="AntiforgeryValidationException"/> with the specified
+        /// exception <paramref name="message"/> and inner exception <paramref name="innerException"/>.
```

Comment:
In general you don't need to mix a paramref and a text description of the parameter. Remember that the parameter refers to the parameter, not the type of the parameter.

I'd just write this as `Creates a new instance of <see cref="AntiforgeryValidationException"/> with the specified exception message and inner exception`.